src/_igraph/graphobject.c

```diff
@@ -13728,6 +13799,22 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "@return: C{True} if it is complete, C{False} otherwise.\n"
    "@rtype: boolean"},
 
+  {"is_clique", (PyCFunction) igraphmodule_Graph_is_clique,
+   METH_VARARGS | METH_KEYWORDS,
+   "is_clique(vertices, directed=False)\n--\n\n"
+   "Decides whether a set of vertices is a clique, i.e. a fully connected subgraph.\n\n"
+   "@param vertices: a list of vertex IDs\n"
+   "@param directed: whether to require mutual connections between vertex pairs\n"
+   "    in directed graphs.\n"
+   "@return: C{True} is the given vertex set is a clique, C{False} if not.\n"},
+
+  {"is_independent_vertex_set", (PyCFunction) igraphmodule_Graph_is_independent_vertex_set,
+   METH_VARARGS | METH_KEYWORDS,
+   "is_independent_vertex_set(vertices)\n--\n\n"
```

Comment:
Same as above: `vertices=None` is supported now and it is optional. The docs should be consistent with the behaviour.

Reply:
Fixed.